MoonDawgArduino/Libraries/CameraControl/Camera.cpp

```diff
@@ -0,0 +1,45 @@
+#include "Camera.h"
+#include <Arduino.h>
+
+CameraControl::CameraControl(){
+    //empty constructor
+}
+
+void CameraControl::cameraSetup(uint8_t horizontalPin, uint8_t verticalPin, uint8_t armPin){
+    horizontalServo.attach(horizontalPin);
+    verticalServo.attach(verticalPin);
+    armServo.attach(armPin);
+    horizontalServo.write(180);
+    verticalServo.write(90);
+    // Initial position for deployment servo
+    // Can change if needed
+    armServo.write(0);
+    // Defines the limits for the horizontal servo
+    // Helps to prevent the cable from getting wrapped around the servo
+    // Can change to higher values if needed
+    horizontalMinAngle = 0;
```

Comment:
why not put this in the constructor

Reply:
Just the constant values for horizontalMinAngle etc since they look like they don't depend on arguments

Reply:
Ah yeah good point. Those were for the 360 servo which got scrapped. They'll be removed along with the horizontalAngle = constrain line in the move horizontal function as that never worked anyways.